HW04  Refactor root pom && add solution for hw04

CombaSoft opened the pull request:
HW04 first attempt.

CombaSoft committed:
Refactor root pom && add solution for hw04

CombaSoft committed:
Remove extra imports added by mistake

vlad-iv commented on hw04/pom.xml:
```diff
@@ -0,0 +1,80 @@
+        <dependency>
+            <groupId>org.springframework.shell</groupId>
+            <artifactId>spring-shell-core</artifactId>
+            <version>3.2.0</version>
```
можно задать версию через переменную spring-shell.version
а лучше удалить, т.к. подключен dependencyManagement

CombaSoft replied:
Удалил. Спасибо, что подсветили!

CombaSoft committed:
Remove specific version for spring-shell-core
 according to Vladimir`s recommendation
 (not required due to DependencyManagement we used).
